packages/dom/src/phrasing-content.js

```diff
@@ -32,7 +32,7 @@ const textContentSchema = {
 	s: {},
 	del: {},
 	ins: {},
-	a: { attributes: [ 'href', 'target', 'rel' ] },
+	a: { attributes: [ 'href', 'target', 'rel', 'id', 'name' ] },
```

Comment:
I think it's not necessary to add `name` here, because the removal of HTML only happens after the phrasing content reducer.

Reply:
I didn't know this, that's great. Just removed it.